lnrpc/routerrpc/router_server.go

```diff
@@ -37,11 +40,15 @@ var (
 
 	// macPermissions maps RPC calls to the permissions they require.
 	macPermissions = map[string][]bakery.Op{
-		"/routerpc.Router/SendPayment": {{
+		"/routerrpc.Router/SendPayment": {{
 			Entity: "offchain",
 			Action: "write",
 		}},
-		"/routerpc.Router/EstimateRouteFee": {{
+		"/routerrpc.Router/LookupPayment": {{
+			Entity: "offchain",
+			Action: "write",
```

Comment:
Why does this require write permissions?

Reply:
Changed to read.